skimage/util/_coord_conv.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+
+
+def xy_to_rc(points=None, image=None):
+    """Convert points' coordinates and/or image from `x, y` to `r, c` convention.
+
```

Comment:
I would add a simple note that the xy and rc are the same orientation and unit just the axis are swapped